bin/seviri_level1c_runner.py

```diff
@@ -21,42 +21,39 @@
 # along with this program.  If not, see <http://www.gnu.org/licenses/>.
 
 
-from seviri_l1c_runner import get_config
-
-from level1c4pps.seviri_hrit import process_one_scan
+from level1c4pps.seviri2pps_lib import process_one_scan  # @UnresolvedImport
+from posttroll.subscriber import Subscribe  # @UnresolvedImport
 from posttroll.publisher import Publish  # @UnresolvedImport
 from posttroll.message import Message  # @UnresolvedImport
 
 import logging
 import sys
 import os
-import six
-from posttroll.subscriber import Subscribe  # @UnresolvedImport
 from multiprocessing import cpu_count
+
+import six
 if six.PY2:
-    from urlparse import urlunsplit
+    from urlparse import urlunsplit  # @UnusedImport
```

Comment:
Do we still need to support python 2 ?

Reply:
Probably not. It was there in the initial files and I just have not seen any reason to remove it.